app/components-react/sidebar/NavTools.tsx

```diff
@@ -302,7 +302,7 @@ function LoginMenuItem(p: {
   return (
     <MenuItem
       title={!isLoggedIn ? menuTitles(menuItem.key) : $t('Log Out')}
-      className={cx(styles.login)}
+      className={cx(styles.login, !isOpen && styles.loginClosed)}
```

Comment:
i'm not sure how cx works with boolean values, we usually do this pattern:
`{ [styles.loginClosed]: !isOpen }`

but if this works too then its no biggie

Reply:
Yeah this works the same but if you really want I can refactor. It just means the entire feature would probably need to be QAed again.

Reply:
sall good if it works

Reply:
Just in case you did want it, I quickly did the edit and made a PR with it to merge into this branch: https://github.com/stream-labs/desktop/pull/4411. Feel free to merge or close as you see fit.